### ISSUE TITLE
Why bias=None in conv2d?

### ISSUE THREAD

**Issue by kxhit** [WangYueFt/dgcnn#37]
Hi! I have some small questions about the code.
1. I noticed that bias is set to None in both PointNet and dgcnn. I'm curious why we do that? I can't find the explanation in these papers.
2. Do we need to apply batch normalization on the input (before conv1)?

Thanks a lot if someone could help me!!!

**Comment by WangYueFt** [comment]
Because we use batchnorm, whether using the bias won't make any difference.
